packages/core/src/web/bridges/hooks/use-bridge.ts

```diff
@@ -25,8 +25,8 @@ const DEFAULT_VALUE_FOR_MCP_APP_BRIDGE: BridgeInterface = {
 
 const getExternalStore = <K extends keyof BridgeInterface>(
   key: K,
-  defaultValue: BridgeInterface[K] = DEFAULT_VALUE_FOR_MCP_APP_BRIDGE[key],
 ): BridgeExternalStore<K> => {
+  const defaultValue = DEFAULT_VALUE_FOR_MCP_APP_BRIDGE[key];
```

Comment:
Fine by me, but in the #141 I'll move it directly in the external store definition, as it'll reduce type safety declaration.

Reply:
In the `getServerSnapshot` implementation :) ? Agreed that it'll be better